solutions/day_6/solution.py

```diff
@@ -0,0 +1,41 @@
+from typing import List
+
+TEXT_FILE_LOCATION = "input.txt"
+ORD_VAL_SET = set()
+
+
+def read_stream_part1(list_of_ord_chars: List[int]) -> int:
+
+    for ord_val_index in range(0, len(list_of_ord_chars)):
+        next_group_of_ords = list_of_ord_chars[ord_val_index : ord_val_index + 4]
+        set_of_next_set_of_ords = set(
+            list_of_ord_chars[ord_val_index : ord_val_index + 4]
+        )
+        if sorted(next_group_of_ords) == sorted(list(set_of_next_set_of_ords)):
+            return ord_val_index + 4
+
+
+def read_stream_part2(list_of_ord_chars: List[int]) -> int:
```

Comment:
This function is identical to the one in part 1, except with `14` instead of `4`. I'd make a helper function and give it `n=4` for `read_stream_part1` and `n=14` for `read_stream_part2`.